spring-boot-parent/src/checkstyle/checkstyle-header.txt

```diff
@@ -1,5 +1,5 @@
 ^\Q/*\E$
-^\Q * Copyright \E20\d\d\-20\d\d\Q the original author or authors.\E$
+^\Q * Copyright \E20\d\d(\-20\d\d)?\Q the original author or authors.\E$
```

Comment:
Interesting - under VMware, legal told us that new files should only contain the year it was created.